### PR TITLE
Listed Thread-Names in top running method list

### PR DESCRIPTION
**Pull request by GaneshSPatil**
In the Top Running Methods list, list thread names per top method and link them all to the stack traces list.

### CODE REVIEW

File: stylesheet.css [hunk 1]
```diff
@@ -55,6 +55,10 @@ div.synchronizer {
     margin-top: 0.25em;
 }
 
+#RUNNING td {
```
Comment (walles) [inline stylesheet.css]: What's `RUNNING` used for? Can't see it referenced anywhere.

File: analyze.js [hunk 2]
```diff
@@ -803,21 +803,20 @@ function Analyzer(text) {
         for (var i = 0; i < countedStrings.length; i++) {
             var countedString = countedStrings[i];
 
+          var ids = countedString.sources.map(function(source){
```
Comment (walles) [inline analyze.js]: This section is mis-indented, could you please fix the indentation?

**Comment by walles** [comment]
Hi @GaneshSPatil!

How have you have tested this change?

  Regards /Johan

**Comment by GaneshSPatil** [comment]
Hey @walles -- sorry for getting back so late,
fixed the comments that you mentioned on the PR
i have updated the PR with new changes

**Comment by walles** [comment]
Sorry for the silly long delay Ganesh, thanks for the PR!

**Pull request merged**